### DISCUSSION TITLE
"message": "Il file specificato non esiste."

### DISCUSSION THREAD

**Discussion by kabottin0** [fattureincloud/api#143] [Q&A] [answered]
Salve a tutti,
come da titolo nella pagina di API Reference inserendo il mio company_id e l'id del metodo di pagamento relativo al mio cliente, non riesce a creare il documento ritornando questo messaggio: 
{
  "error": {
    "message": "Il file specificato non esiste.",
    "validation_result": null
  }
}

Togliendo la chiave attachment_token che penso riguardava il file "non esistente" ritorna un altro tipo di errore:
{
  "error": {
    "message": "Cannot perform the operation."
  }
}

Qualcuno di voi ha già avuto una situazione simile o ha già risolto?
Grazie per l'aiuto.

**Comment by valmoz** [comment, marked as answer]
Buongiorno,
per la prima parte del problema è esatto, sta usando un attachment token relativo a un documento inesistente o non caricato da lei.

Per indagare il problema "cannot perform the operation" le consiglio di contattare il supporto clienti, aggiungendo il link a questa discussione. In questo modo potremo ricevere dettagli sensibili come il company id che non possono essere condivisi tramite Community.

Grazie
Mauro

**Reply by kabottin0** [reply]
ti ringrazio per la disponibilità

**Reply by valmoz** [reply]
Buongiorno @kabottin0 è riuscito a risolvere il problema?
